enterprise/urls.py

```diff
@@ -74,21 +74,29 @@
         include('consent.urls'),
         name='consent'
     ),
-    path(
-        'cornerstone/',
-        include('integrated_channels.cornerstone.urls'),
-        name='cornerstone'
-    ),
-    path(
-        'canvas/',
-        include('integrated_channels.canvas.urls'),
-        name='canvas',
-    ),
-    path(
-        'blackboard/',
-        include('integrated_channels.blackboard.urls'),
-        name='blackboard',
-    ),
+]
+
+# When enabled, these URLs will be registered and take precedence over new plugin URLs
+if getattr(settings, 'ENABLE_LEGACY_INTEGRATED_CHANNELS', True):
+    urlpatterns += [
+        path(
+            'cornerstone/',
+            include('integrated_channels.cornerstone.urls'),
+            name='cornerstone'
+        ),
+        path(
+            'canvas/',
+            include('integrated_channels.canvas.urls'),
+            name='canvas',
+        ),
+        path(
+            'blackboard/',
+            include('integrated_channels.blackboard.urls'),
+            name='blackboard',
+        ),
+    ]
+
```

Comment:
Curious to understand why are we only enabling it for just these 3 channels?

Reply:
All of these are getting mounted at the root for example: `http://localhost:18000/cornerstone/course-list` so we are moving them behind a feature flag so that they don't cause trouble in our migration work. FYI from these 3 channels only cornerstone has an endpoint and the rest are empty